src/components/Tokens/TokenDetails/About.tsx

```diff
@@ -108,7 +111,12 @@ export function AboutSection({ address, chainId, description, homepageUrl, twitt
           name={chainId === SupportedChainId.MAINNET ? 'Etherscan' : 'Block Explorer'}
           link={`${baseExplorerUrl}${address === 'NATIVE' ? '' : 'address/' + address}`}
         />
-        <Resource name="More analytics" link={`https://info.uniswap.org/#/tokens/${address}`} />
+        <Resource
+          name="More analytics"
+          link={`https://info.uniswap.org/#/${
```

Comment:
this isn't going to work since some `labels` are not valid strings to be used for the URL here, like `Celo Alfajores` or `BNB Chain` 

`getChainInfo` returns `infoLink` for the prefix you want - can you just use that here instead?